docs/data/charts/export/ExportChartAsImage.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import { ChartProApi } from '@mui/x-charts-pro/ChartContainerPro';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Radio from '@mui/material/Radio';
+import TextField from '@mui/material/TextField';
+import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
```

Comment:
Nitpick: I tend to group all import specific to the library at the bottom. It goes from general imports (react, mui/material) to demo specific.

```suggestion
import FormControl from '@mui/material/FormControl';
import FormLabel from '@mui/material/FormLabel';
import RadioGroup from '@mui/material/RadioGroup';
import FormControlLabel from '@mui/material/FormControlLabel';
import Radio from '@mui/material/Radio';
import TextField from '@mui/material/TextField';
import { LineChartPro } from '@mui/x-charts-pro/LineChartPro';
import { ChartProApi } from '@mui/x-charts-pro/ChartContainerPro';
```